go.mod

```diff
@@ -173,6 +173,8 @@ replace (
 	github.com/openshift/client-go => github.com/openshift/client-go v0.0.0-20210112165513-ebc401615f47
 	github.com/operator-framework/operator-lifecycle-manager => github.com/operator-framework/operator-lifecycle-manager v0.0.0-20190128024246-5eb7ae5bdb7a
 
+	golang.org/x/crypto => github.com/openshift/golang-crypto v0.33.1-0.20250310193910-9003f682e581
```

Comment:
Can you add a comment here that this replace is required to avoid bumping the golang version?

Reply:
addressed thia review, thanks @0xFelix!

Reply:
addressed this review, thanks @0xFelix!